L1Trigger/L1TCalorimeter/src/firmware/Stage1Layer2EtSumAlgorithmImpHW.cc

```diff
@@ -65,87 +64,63 @@ void l1t::Stage1Layer2EtSumAlgorithmImpHW::processEvent(const std::vector<l1t::C
   std::vector<double> regionPUSParams = params_->regionPUSParams();
   RegionCorrection(regions, subRegions, regionPUSParams, regionPUSType);
 
-  // double towerLsb = params_->towerLsbSum();
-  // int jetSeedThreshold = floor( params_->jetSeedThreshold()/towerLsb + 0.5);
-  // // ----- cluster jets for repurposing of MHT phi (use if for angle between leading jet)
-  // std::vector<l1t::Jet> *unCorrJets = new std::vector<l1t::Jet>();
-  // std::vector<l1t::Jet> * unSortedJets = new std::vector<l1t::Jet>();
-  // std::vector<l1t::Jet> * SortedJets = new std::vector<l1t::Jet>();
-  // slidingWindowJetFinder(jetSeedThreshold, subRegions, unCorrJets);
-
-  //if jetCalibrationType is set to None in the config
-  // std::string jetCalibrationType = params_->jetCalibrationType();
-  // std::vector<double> jetCalibrationParams = params_->jetCalibrationParams();
-  // JetCalibration(unCorrJets, jetCalibrationParams, unSortedJets, jetCalibrationType, towerLsb);
-
-  // SortJets(unSortedJets, SortedJets);
-  // int dijet_phi=DiJetPhi(SortedJets);
-
-  for(std::vector<CaloRegion>::const_iterator region = subRegions->begin(); region != subRegions->end(); region++) {
-    if (region->hwEta() < etSumEtaMinEt || region->hwEta() > etSumEtaMaxEt) {
-      continue;
-    }
-
-    //double regionET= regionPhysicalEt(*region);
-    int regionET = region->hwPt();
+  std::vector<SimpleRegion> regionEtVect;
+  std::vector<SimpleRegion> regionHtVect;
 
-    if(regionET >= etSumEtThresholdEt){
-      sumET += regionET;
-      sumEx += (((double) regionET) * cosPhi[region->hwPhi()]);
-      sumEy += (((double) regionET) * sinPhi[region->hwPhi()]);
-    }
-  }
-
-  for(std::vector<CaloRegion>::const_iterator region = subRegions->begin(); region != subRegions->end(); region++) {
-    if (region->hwEta() < etSumEtaMinHt || region->hwEta() > etSumEtaMaxHt) {
-      continue;
+  // FIXME: hwPt() is ambiguous as to Et/Ht, need to properly fill
+  // hwEtEm() and hwEtHad() and use approriate RegionCorrection()
+  // Also, need to confirm thresholds will be hardware values not physical
```

Comment:
Will [RegionCorrection](https://github.com/cms-l1t-offline/cmssw/blob/EtSumAlg-devel/L1Trigger/L1TCalorimeter/src/PUSubtractionMethods.cc#L79) be using `CaloRegion::hwPt()` going forward? (as opposed to `hwEtEm()`)  Are we going to correct HT as well?